recipes/pytest-bdd/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "pytest-bdd" %}
+{% set version = "3.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "17dfc2b65c275641a4e76fa1f913ab737604815c275dc2afe7133956c6ab6743" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```

As these are only used once:
```suggestion
  sha256: 17dfc2b65c275641a4e76fa1f913ab737604815c275dc2afe7133956c6ab6743
```